bin/hdfcoinc/pycbc_page_coinc_snrchi

```diff
@@ -133,7 +137,7 @@ ax.set_yscale('log')
 try:
     cb = pylab.colorbar()
     cb.set_label(coloring[args.colorbar_choice][1], size='large')
-except TypeError:
+except (TypeError, ZeroDivisionError):
```

Comment:
Perhaps you need a comment here to say what causes the ZeroDivisionError and/or how it does so? 
Is ZeroDivisionError raised when there is exactly one found inj?

Reply:
I think this is different versions of numpy raising different errors (I couldn't get a TypeError). But the ZeroDivisionError is raised when there are no injection triggers (as the comment below states). I think the code is trying to find the upper and lower bounds of the [logarithmic] colorbar and is failing with no data.

Reply:
right .. do you think it would be possible to see what happens if there is exactly 1 found inj, since the other line that is checking for 'not enough' was 
`if len(coloring[args.colorbar_choice][0]) < 2:`

Reply:
I've verified that the different errors are caused by different numpy (or matplotlib or ...) versions.

For the case of exactly one trigger the code seems to run, although the colorbar may not be so useful. I've changed the code to check for len == 0 now though.

Also be aware that matplotlib's behaviour is dependent on the backend in use (normally 'Agg', but even that isn't identical across operating systems). The point is that there still may be other ways that 0 injections could cause failures in different environments/OSes. For now let's just play "whack-a-mole" with this kind of thing.